### PR TITLE
MF - 1758 - Sync With Benchmark Testing

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
1. Add trace ratio for traces `MF_JAEGER_TRACE_RATIO` i.e. don't always trace every request. 1.0 means always sample while 0.0 means never sample. This has proven useful for benchmark testing as when the ratio is 1.0 the Jaeger container uses a lot of memory
2. Add Postgres max connection `MF_POSTGRES_MAX_CONNECTIONS`. This is to increase the connected clients to Postgres i.e. previously a max of 100 clients was not ideal for testing

### Which issue(s) does this PR fix/relate to?
Related to #1758 

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### Notes
Linked with https://github.com/mainflux/benchmark/pull/10

### CODE REVIEW

**Review by SammyOina — APPROVED** [review]

File: docker/.env [hunk 1]
```diff
@@ -60,10 +60,14 @@ MF_JAEGER_FRONTEND=16686
 MF_JAEGER_COLLECTOR=14268
 MF_JAEGER_CONFIGS=5778
 MF_JAEGER_URL=http://jaeger:14268/api/traces
+MF_JAEGER_TRACE_RATIO=1.0
 
 ## Call home
 MF_SEND_TELEMETRY=true
 
+## Postgres
+MF_POSTGRES_MAX_CONNECTIONS=100000
```
Comment (arvindh123) [inline docker/.env]: Why we should have 100,000 connections by default ?

**Review by arvindh123 — CHANGES_REQUESTED** [review]
Expect `POSTGRES_MAX_CONNECTIONS` limit everything look good.

Some thing can we have like this below 

In .env
```diff
--  MF_POSTGRES_MAX_CONNECTIONS=100
```
In docker-compose.yaml
```diff
--       MF_POSTGRES_MAX_CONNECTIONS: ${MF_POSTGRES_MAX_CONNECTIONS}
++     MF_POSTGRES_MAX_CONNECTIONS: ${MF_POSTGRES_MAX_CONNECTIONS:-100}}
```

File: docker/.env (hunk 1 above)
Comment (rodneyosodo) [reply docker/.env]: Reduced to 100 i.e default

**Review by arvindh123 — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**